services/users/services.go

```diff
@@ -0,0 +1,75 @@
+/*This file contains the required businees logic implementation. Internally it is using the
+  CRUD functinality provided in the stores package*/
+
+package users
+
+import (
+	"errors"
+
+	"github.com/arohanzst/user-curd/models"
+	"github.com/arohanzst/user-curd/services"
+	"github.com/arohanzst/user-curd/stores"
+)
+
+type User struct {
+	u stores.User
+}
+
+func New(u stores.User) services.User {
+	return &User{u}
+}
+
+//Creates a New User
+func (st *User) Create(value models.User) (int64, int64, error) {
+	lastInsertId, affect, err := st.u.Create(value)
+
+	if err != nil {
+		return 0, -1, errors.New("Given user could not be created.")
+	}
+
+	return lastInsertId, affect, nil
```

Comment:
Return the inserted data by using the ReadById function with the lastInsertId to fetch the inserted data then return that.
no need to return lastInsertId.